src/lib/operators/delete.hpp

```diff
@@ -11,15 +11,11 @@
 namespace opossum {
 
 /**
- * Operator that deletes a number of rows from one table.
- * Expects a table with one chunk referencing only one table which
- * is passed via the AbstractOperator in the constructor.
- *
- * Assumption: The input has been validated before.
```

Comment:
Why did you remove the assumption?

Reply:
Accidentally. But it makes a nice point: Verbose commenting leads to the important parts being overlooked and potentially deleted.